.jenkins/pod.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Pod
+spec:
+  nodeSelector:
+    jenkins_worker: "true"
+
+  volumes:
+  - name: cache
+    hostPath:
+      path: /opt/kubernetes/cache
+  - name: jenkins-credentials
+    persistentVolumeClaim:
+      claimName: jenkins-credentials
+
+  containers:
+  - name: jnlp
+    terminationGracePeriodSeconds: 0
+
+  - name: build
+    image: gtx-docker-jenkinsbuilds.docker.apa-it.at/gentics/jenkinsbuilds/nodejs-worker:1.0
+    workingDir: '/home/jenkins'
+    imagePullPolicy: Always
+    terminationGracePeriodSeconds: 0
+    command:
+      - cat
+    tty: true
+    resources:
+      requests:
+        cpu: 1
+        memory: 1Gi
+    securityContext:
+      privileged: true
+    volumeMounts:
+    - mountPath: /home/jenkins/.m2/repository
+      name: cache
+      subPath: maven
+    - mountPath: /home/jenkins/.gpg-keys
+      name: jenkins-credentials
```

Comment:
Check whether you could add:

Host: /opt/kubernetes/cache/npm/_cacache
Local Mount: /home/jenkins/.npm/_cacache